sktime/markov/hmm/output_model.py

```diff
@@ -286,6 +289,17 @@ def sample(self, observations_per_state: List[np.ndarray]) -> None:
         """
         # todo why ignore observation states w/o counts in a hidden state parameter sample?
         _bindings.discrete.sample(observations_per_state, self.output_probabilities, self.prior)
+        # from numpy.random import dirichlet
+        # N, M = self._output_probabilities.shape  # nstates, nsymbols
+        # for i, obs_by_state in enumerate(observations_per_state):
+        #     # count symbols found in data
+        #     count = np.bincount(obs_by_state, minlength=M).astype(float)
+        #     # sample dirichlet distribution
+        #     count += self.prior[i]
+        #     positive = count > 0
+        #     # if counts at all: can't sample, so leave output probabilities as they are.
+        #     self._output_probabilities[i, positive] = dirichlet(count[positive])
+        # self._output_probabilities /= np.sum(self._output_probabilities, axis=1)[:, None]
```

Comment:
what's this?

Reply:
old code, can be removed

Reply:
lots of old code i only marked the first instance!